first-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+PWD=$(pwd)
+pkgname=${PWD##*/}
+
+git init
```

Comment:
If we want to run this inside this repository, we should add git init <directory>. Otherwise git init will grab the git repository from the parent directory.